lms/djangoapps/courseware/views.py

```diff
@@ -1050,6 +1051,16 @@ def _progress(request, course_key, student_id):
     # checking certificate generation configuration
     show_generate_cert_btn = certs_api.cert_generation_enabled(course_key)
 
+    if is_credit_course(course_key):
+        requirement_statuses = get_credit_requirement_status(course_key, student.username)
+        credit_course = {
+            'show_eligibility_table': True,
+            'eligibility_status': "eligible",
```

Comment:
Why hard coded "eligible" status. Should it not be checked that has user passed all the requirements or not?